draft-ietf-scitt-architecture.md

```diff
@@ -559,12 +559,15 @@ The Transparency Service MAY verify the Statement payload format, content and ot
   Custom Signed Statements are evaluated given the current Transparency Service state and the entire Envelope, and may use information contained in the attributes of named policies.
 1. **Register the Signed Statement** to the append-only log.
 1. **Return the Receipt**, which MAY be asynchronous from registration.
+When asynchronous, the Transparency Service MUST return an identifier used to retrieve the Receipt.
```

Comment:
Where is this identifier defined? What does it identify? There is a potential ambiguity between identifying a registration event (later authenticated by the receipt) or more loosely just a signed statement, which may then be registered later, or refer to an earlier registration, possibly subsumed by others. Unless/until we agree on registration-event identifiers, I suggest we cut this sentence.

Reply:
in an asynchronous flow like when batch writing, you need an identifier to use to retrieve the receipt when it eventually becomes available,.

Reply:
The simpler way to prevent confusion is to remove this sentence. Why is it required in the arch?

Reply:
Could we write "correlator" instead of identifier to side-step this issue?

Reply:
Could we fiip the sense and instead say

```suggestion
The Transparency Service MUST be able to provide a receipt for all registered statements. A receipt for a signed statement MAY be provided asyncrhously```

I think this just brings what has been said previously closer together but the adjacency makes it easire to digest.

This could be tightened by adding

"Asynchronously obtained receipts MUST be demonstrably equivelent to a receipt obtained at the point of registration"

Reply:
I think this just brings what has been said previously closer together but the adjacency makes it easire to digest.

This could be tightened by adding

"Asynchronously obtained receipts MUST be demonstrably equivelent to a receipt obtained at the point of registration"